test/delete.submission.browser.js

```diff
@@ -27,13 +27,13 @@ test('Delete a Submission', async t => {
   await t
     .click(author.orcidPrefill)
     .expect(author.firstNameField.value)
-    .eql('Tamlyn')
+    .eql('Josiah')
     .expect(author.secondNameField.value)
-    .eql('Rhodes')
+    .eql('Carberry')
     .expect(author.emailField.value)
-    .eql('f72c502e0d657f363b5f2dc79dd8ceea')
+    .eql('j.carberry@orcid.org')
```

Comment:
@diversemix will this need to be obfuscated?

Reply:
This is a conventional fake name, don't worry about that

Reply:
That is a good point - we've tried to avoid committing anything that looks like an email address to make it easier to identify emails that should *not* be committed